CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## v2.9.0 / 2023-05-23
+
+The changes mentioned below are only the user-facing ones. For a list of the complete set of changes, refer the changelog associated with the release tag.
+
+Note: The experimental VerticalPodAutoscaler metrics are no longer supported, and have been removed.
+
+* [BUGFIX] Adhere to OTel-Prometheus standard for labels #2004 @rexagod
```

Comment:
Was this possibly a breaking change for experimental _label metrics and should be called out as well?